apps/molgenis-components/src/components/tables/Pagination.vue

```diff
@@ -15,23 +15,23 @@
         <a
           class="page-link"
           href="#"
-          @click.prevent="emitValue(Math.max(value - 1, 1), isFirstPage)"
```

Comment:
I dont see why.. but pagination in de preview

https://preview-emx2-pr-1778.dev.molgenis.org/apps/molgenis-components/#/component/Pagination

Starts at `-9 of 0`

here https://emx2.dev.molgenis.org/apps/molgenis-components/#/component/Pagination
it starts with `1 of 10`

Reply:
that was a bug, on created() should check the modelValue, did check old value.